### ISSUE TITLE
CLI records -h doesn't work

### ISSUE THREAD

**Issue by abbazs** [kennethreitz/records#50]
I'm unable to get the CLI records to display full help with -h or --help option, it throws the following error.
```
PS D:\APPS\Python\3.5> records -h
Traceback (most recent call last):
  File "d:\apps\python\3.5\lib\runpy.py", line 170, in _run_module_as_main
    "__main__", mod_spec)
  File "d:\apps\python\3.5\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "D:\APPS\Python\3.5\Scripts\records.exe\__main__.py", line 9, in <module>
  File "d:\apps\python\3.5\lib\site-packages\records.py", line 318, in cli
    arguments = docopt(cli_docs)
  File "d:\apps\python\3.5\lib\site-packages\docopt.py", line 575, in docopt
    extras(help, version, argv, doc)
  File "d:\apps\python\3.5\lib\site-packages\docopt.py", line 481, in extras
    print(doc.strip("\n"))
  File "d:\apps\python\3.5\lib\encodings\cp437.py", line 19, in encode
    return codecs.charmap_encode(input,self.errors,encoding_map)[0]
UnicodeEncodeError: 'charmap' codec can't encode character '\u2122' in position 23: character maps to <undefined>
PS D:\APPS\Python\3.5> records --help
Traceback (most recent call last):
  File "d:\apps\python\3.5\lib\runpy.py", line 170, in _run_module_as_main
    "__main__", mod_spec)
  File "d:\apps\python\3.5\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "D:\APPS\Python\3.5\Scripts\records.exe\__main__.py", line 9, in <module>
  File "d:\apps\python\3.5\lib\site-packages\records.py", line 318, in cli
    arguments = docopt(cli_docs)
  File "d:\apps\python\3.5\lib\site-packages\docopt.py", line 575, in docopt
    extras(help, version, argv, doc)
  File "d:\apps\python\3.5\lib\site-packages\docopt.py", line 481, in extras
    print(doc.strip("\n"))
  File "d:\apps\python\3.5\lib\encodings\cp437.py", line 19, in encode
    return codecs.charmap_encode(input,self.errors,encoding_map)[0]
UnicodeEncodeError: 'charmap' codec can't encode character '\u2122' in position 23: character maps to <undefined>
```

**Comment by kennethreitz** [comment]
Awe, this must be an incompatibility with your terminal. Currently, the output includes a few emoji for fun.

I'll remove them, sorry about that.

**Comment by kennethreitz** [comment]
@abbazs just released v0.4.3, which should fix this! I apologize for the bug.

**Comment by emre** [comment]
:+1:
